flow/Platform.actor.cpp

```diff
@@ -2428,23 +2423,23 @@ static bool deletedFile() {
 	return true;
 }
 
-bool deleteFile(std::string const& filename) {
+bool deleteFile(std::filesystem::path path) {
 	INJECT_FAULT(platform_error, "deleteFile"); // file deletion failed
 #ifdef _WIN32
```

Comment:
std::filesystem has the delete no need to distinguish between OS

Reply:
I think latest commit fixes this, feel free to correct me

Reply:
Perfect!